server/models/game.go

```diff
@@ -30,12 +32,21 @@ type Player struct {
 	Points uint64 `json:"points"`
 }
 
+type Prompts struct {
+	Author     string
+	Group      string
+	Noun       string
+	Adjective1 string
```

Comment:
same nit: maybe we can have adjectives as a string array? No big thing though, feel free to leave as is for now

Reply:
makes sense